Gulpfile.ts

```diff
@@ -608,10 +608,19 @@ gulp.task("LKG", "Makes a new LKG out of the built js files", ["clean", "dontUse
     return runSequence("LKGInternal", "VerifyLKG");
 });
 
+gulp.task("typemock", () => {

```

Comment:
You should probably add either a jake task that mirror this, or a jake task that shells out to this; otherwise the `jake tests` task will fail if you have yet to build `typemock`, no? At least until we remove `jake`, which is something I, personally, would be happy to entertain again. The team has had, like, 2 years to adopt new habits. 🐱 Hmmm... if outright removing it is too much right now (just because muscle/task memory), then maybe swapping all the jakefile's tasks with ones that shellout to `gulp` would be appropriate... Gah, that's a topic for another issue. In any case, this typemock-building functionality should probably be accessible via `jake` in some way.